relay-redis/src/config.rs

```diff
@@ -115,3 +159,30 @@ server: "redis://127.0.0.1:6379"
         }
     }
 }
+
+#[test]
+fn test_redis_cluster_nodes_opts() {
```

Comment:
Is there any reason to not add a module for tests?

Reply:
just wrong copy/paste, fixed it now